pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -152,6 +147,39 @@ func (uc *UpgradeCluster) getClusterNodeStatus(subscriptionID uuid.UUID, resourc
 	return nil
 }
 
+func (uc *UpgradeCluster) upgradable(vmOrchestratorTypeAndVersion string) error {
+	arr := strings.Split(vmOrchestratorTypeAndVersion, ":")
+	if len(arr) != 2 {
+		return fmt.Errorf("Unsupported orchestrator tag format %s", vmOrchestratorTypeAndVersion)
+	}
+	currentVer := arr[1]
+	arr = strings.Split(currentVer, ".")
+	if len(arr) != 3 {
+		return fmt.Errorf("Unsupported orchestrator version format %s", currentVer)
+	}
+	currentRel := fmt.Sprintf("%s.%s", arr[0], arr[1])
+
+	cs := &api.ContainerService{
```

Comment:
Why does it need to construct a whole new ContainerService model to get GetOrchestratorVersionProfile

Reply:
addressed